lib/web/css/source/lib/_utilities.less

```diff
@@ -22,6 +22,13 @@
     @fontValue: @_value;
 }
 
+.lib-sticky-nav() {
```

Comment:
This is not specific to navigation, can be renamed to e.g `lib-sticky`.

I'd probably also add `z-index` and `top` as arguments to the signature, rather than blindly throwing a large z-index onto it.